src/Components/Components/src/Forms/InputComponents/InputBase.cs

```diff
@@ -71,7 +71,7 @@ protected T CurrentValue
                 if (hasChanged)
                 {
                     Value = value;
-                    ValueChanged?.Invoke(value);
+                    _ = ValueChanged.InvokeAsync(value);
```

Comment:
Because this is fire and forget, it's possible for no one to see the error if one happens. 

I've considered in the past whether we should have a `HandleError(Task)` method on `ComponentBase` in case you find yourself in one of these situations. 

It doesn't seem in scope for this PR? but thoughts in general?

Reply:
Yes we should avoid unobserved tasks in general.

Reply:
Yes, I also agree we should explicitly handle this somehow, and the `ComponentBase.HandleError` approach sounds promising.

That's well out of scope for this PR, of course!

Also it's not a scenario that's going to affect conventional uses of `bind-*`, since with those, the "setter" code we generate is always synchronous anyway. You couldn't return a `Task` to the system even if you wanted to.